app/views/participant_bulk_actions/add_note.html.erb

```diff
@@ -0,0 +1,23 @@
+<h2>Add Note</h2>
+<p>Add a note for <%= raw(@participants.collect{ |p| link_to p.fullname.html_safe, p }.to_sentence) %>.</p>
```

Comment:
I think this is possible XSS, as html_safe just changes the state of the string to be "html safe", but p.fullname hasn't gone through any sanitizing/escaping.